etc/checkstyle-suppressions.xml

```diff
@@ -57,6 +57,10 @@
     <suppress checks="IllegalImport"
             files="DerUtils\.java"/>
 
+    <!-- Metrics Prometheus output generation depends on numerous settings -->
+    <suppress checks="ParameterNumber"
+            files="MetricImpl\.java"/>
+
```

Comment:
Can I ask release manager @barchetta for review?

Reply:
@barchetta Just for context, this is the same change as with the 2.x change you previously reviewed.

Reply:
LGTM